src/Application/Ignore/IgnoredErrorHelperResult.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigStan\Application\Ignore;
+
+use Symfony\Component\Filesystem\Path;
+use TwigStan\Application\TwigStanError;
+use TwigStan\Twig\SourceLocation;
+
+final class IgnoredErrorHelperResult
```

Comment:
I feel this class can be merged together with IgnoredErrorHelper.

And then call it `IgnoredErrorProcessor`.